.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ on:
   release:
     types: [ published ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, version/v7.2.1 ]
```

Comment:
if you changed the base-branch to main, you dont needto include this one

Reply:
Reverted to main only.